packages/usa-pagination/src/usa-pagination.twig

```diff
@@ -2,8 +2,8 @@
 {% from "@components/usa-pagination/src/includes/_pagination-arrow.twig" import paginationArrow %}
 
 {% set overflow %}
-  <li class="usa-pagination__item usa-pagination__overflow" role="presentation">
```

Comment:
pa113 was flagging the following error:
```
<ul> and <ol> must only directly contain <li>, <script> or
  <template> elements
  (https://dequeuniversity.com/rules/axe/4.5/list?application=axeAPI)

  (#pagination-preview-content > nav:nth-child(2) > ul)

  <ul class="usa-pagination__list"> <li
  class="usa-pagination_...</ul>
```

I am not able to replicate this error in Axe.